docs/dependency-updates.md

```diff
@@ -52,100 +56,129 @@ When updating Node.js, keep in mind:
-  - `./package.json`
-  - `./web-api/runtimes/puppeteer/package.json`
-3. Manually update the images in:
-  - `./Dockerfile`
-  - `./web-api/runtimes/puppeteer/Dockerfile`
-3. Manually update the Node.js version in:
-  - `./.circleci/config.yml`
-4. Manually update DAWSON's GitHub Actions YAML files.
-  - **Note:** These files will point to `.nvmrc` in a future update.
-5. Update the node version used by our lambdas. 
-  - `web-api/terraform/modules/lambda/lambda.tf`
-  - `web-api/terraform/modules/api/layers.tf`
+1. Manually update the `"engines"` property in:
+   - `./package.json`
+   - `./web-api/runtimes/puppeteer/package.json`
+1. Manually update the images in:
+   - `./Dockerfile`
+   - `./web-api/runtimes/puppeteer/Dockerfile`
+1. Manually update the Node.js version in:
+   - `./.circleci/config.yml`
+1. Manually update DAWSON's GitHub Actions YAML files.
+   - **Note:** These files will point to `.nvmrc` in a future update.
+1. Update the node version used by our lambdas.
+   - `web-api/terraform/modules/lambda/lambda.tf`
+   - `web-api/terraform/modules/api/layers.tf`
 
 #### 2.2 Update `Dockerfile` as needed
 
-Check if there are updates to any the following in the main `Dockerfile`. Changing the `Dockerfile` requires publishing a new ECR image which is used as the docker image in CircleCI.
+Check if updates are necessary for the main `Dockerfile`. We base this image on `cypress/browsers`, a debian linux image that contains the latest (headless) versions of Chrome, Firefox, and Edge.
 
-- `terraform`: check for a newer version on the [Terraform site](https://developer.hashicorp.com/terraform/install).
-  - Check for the version number and compare with the current version
-  - Change the version of the `terraform.zip` that we retrieve in `./Dockerfile`
-  - Change the version in `scripts/verify-terraform-version.sh`
-- `aws-cli`: check for a newer version on [AWS CLI](https://github.com/aws/aws-cli/tags) and use the latest version you can find for 2.x, replace it in the DockerFile
-- `docker cypress/base image`: [Check DockerHub](https://hub.docker.com/r/cypress/browsers/tags?page=1&name=node-22) if an update is available for the current node version the project is using.
+- Base image - `cypress/browsers`
+   - Check [DockerHub](https://hub.docker.com/r/cypress/browsers/tags?page=1&name=node-22) if an update is available for the current node version the project is using
+   - Change the `FROM` line in the `Dockerfile` to use the new version
+- `terraform`
+   - Check the [Terraform site](https://developer.hashicorp.com/terraform/install) if an update is available
+   - Change the version of the `terraform.zip` that we retrieve in `./Dockerfile`
+   - Change the version in `scripts/verify-terraform-version.sh`
+- `aws-cli`
+   - Check for the latest 2.x version of the [AWS CLI](https://github.com/aws/aws-cli/tags)
+   - Change the version of the `awscliv2.zip` that we retrieve in `./DockerFile`
```

Comment:
Clarified how the base image works and moved the step to update it to come before the steps to update terraform and aws-cli.